Library/Formula/dropbox-uploader.rb

```diff
@@ -0,0 +1,16 @@
+class DropboxUploader < Formula
+  desc "Bash script for interacting with Dropbox"
+  homepage "http://www.andreafabrizi.it/?dropbox_uploader"
+  url "https://github.com/andreafabrizi/Dropbox-Uploader/archive/0.16.tar.gz"
+  sha256 "738aebdb57f75e9033cb9bb319b14fe1c3b471f951180acb48631e9810ebf1c3"
+
+  depends_on "curl"
+
+  def install
+    bin.install "dropbox_uploader.sh"
+  end
+
+  test do
+    system "true"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!